src/main/java/com/example/accountingapp/service/ProductService.java

```diff
@@ -0,0 +1,10 @@
+package com.example.accountingapp.service;
+
+import com.example.accountingapp.dto.ProductDTO;
+
+import java.util.List;
+
+public interface ProductService {
```

Comment:
Compiler error... we have to add serviceImpl class to run it properly

Reply:
Please try now!